suite-native/intl/src/hooks/useSystemLocaleTag.ts

```diff
@@ -3,33 +3,25 @@ import { AppState, AppStateStatus } from 'react-native';
 
 import { getLocales } from 'expo-localization';
 
-import { NativeLocale, isSupportedNativeLocale } from '@suite-common/suite-types';
+const readSystemLocale = () => {
```

Comment:
Would this be more appropriate naming?
```suggestion
const readSystemLocaleTag = () => {
```

Reply:
fixed in [0fe8ef6](https://github.com/trezor/trezor-suite/pull/22614/commits/0fe8ef64c46c3fb468c1321900c3a9077a3a858a)